### PR TITLE
chore: Remove stale values from docker-compose.yml

### PR DESCRIPTION
**Pull request by bo0tzz**

### CODE REVIEW

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -40,10 +40,6 @@ services:
       - model-cache:/cache
     env_file:
       - .env
-    environment:
-      - NODE_ENV=production
```
Comment (alextran1502) [inline docker/docker-compose.yml:44]: we will need this one to toggle hot-reload in development

Comment (bo0tzz) [reply docker/docker-compose.yml:44]: This container doesn't use node anymore ;)

Comment (alextran1502) [reply docker/docker-compose.yml:44]: we still use that environment variable name 😢

Comment (bo0tzz) [reply docker/docker-compose.yml:44]: Gotcha

**Pull request merged**